monitoring/README.md

```diff
@@ -0,0 +1,133 @@
+# Prometheus and Grafana deployment
+
+This deployment uses the [CoreOS Prometheus Operator](https://coreos.com/operators/prometheus/docs/0.15.0/index.html). 
+
+**This monitoring folder contains the following functions:**
+* deploy scripts to:
+    * deploy the Prometheus Operator along with Grafana and Alert Manager and other Helm charts that help monitor GKE.
+    * connect to the Prometheus and Grafana endpoints.
+* exporter-forgerock Helm chart that contains a Helm chart that provides:
+    * configurable ServiceMonitors that define ForgeRock product endpoints to be monitored by Prometheus.
```

Comment:
May be better as: * exporter-forgerock Helm chart that provides configurable ServiceMonitors. The ServiceMonitorst define the ForgeRock Identity Platform component endpoints that are 
monitored by Prometheus.

Reply:
sounds good, changed